pkg/utils/utils.go

```diff
@@ -0,0 +1,305 @@
+/*
+Copyright © 2021 Rasa Technologies GmbH
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package utils
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/go-logr/logr"
+	"github.com/imdario/mergo"
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+	"github.com/txn2/txeh"
+	"golang.org/x/term"
+
+	"github.com/RasaHQ/rasactl/pkg/types"
+)
+
+type NetworkError string
+
+// validName is a regular expression for names.
+// See: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+var validName = regexp.MustCompile(`^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`)
+
+const (
+	NetworkErrorConnectionRefused NetworkError = "ConnectionRefused"
+)
+
+func IsDebugOrVerboseEnabled() bool {
+	if viper.GetBool("debug") || viper.GetBool("verbose") {
+		return true
+	}
+
+	return false
+}
+
+func CheckNetworkError(err error) (NetworkError, error) {
+	switch t := err.(type) {
+	case *url.Error:
+		if t.Op == "Get" {
+			return NetworkErrorConnectionRefused, err
+		}
+	}
+	return "", err
+}
+
+// MergeMaps merges maps into one.
+func MergeMaps(maps ...map[string]interface{}) map[string]interface{} {
+	result := make(map[string]interface{})
+	for _, m := range maps {
+		if err := mergo.Map(&result, m, mergo.WithOverride); err != nil {
+			continue
+		}
```

Comment:
Since the loop doesn't have anything else in the body, doing `continue` or not is the same! So checking the error in this case is not changing anything.